aten/src/ATen/function_wrapper.py

```diff
@@ -157,14 +166,36 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 #endif
 }
 """)
+
+C10_TENSOR_METHOD_DEFINITION_UNDERSCORE = CodeTemplate("""\
+inline ${return_type} Tensor::_${api_name}(${method_formals}) const {
+#ifdef USE_STATIC_DISPATCH
+    ${static_dispatch_method_body}
+#else
+    static c10::OperatorHandle op = c10::Dispatcher::singleton().findSchema({"aten::${operator_name}", "${overload_name}"}).value();
+    return op.callUnboxed<${formals_types_with_return}>(${method_actuals});
+#endif
+}
+""")
+
 # add a method declaration in Functions.h
 FUNCTION_DECLARATION = CodeTemplate("""\
 static inline ${return_type} ${api_name}(${formals_with_defaults});
 """)
+FUNCTION_DECLARATION_UNDERSCORE = CodeTemplate("""\
+static inline ${return_type} _${api_name}(${formals_with_defaults});
+""")
+COLLAPSED_FUNCTION_DECLARATION = CodeTemplate("""\
```

Comment:
In the context of this PR, it is clear what "collapsed" means, but it would probably be helpful if there was a brief English description of what collapsed functions are.